### ISSUE TITLE
Archiving code snippets

### ISSUE THREAD

**Issue by paaljoachim** [sheabunge/code-snippets#153]
I do not feel like deleting the default code snippets as I believe they can give some tips at a later time, but I want to remove them out of the All Snippets screen to an archive page or a trash can page. 

- Removing none needed code snippets into another screen. As I am not ready to delete these.
- Experimentation code snippets could also be moved into another screen.

The result is a cleaner All Snippets list.

**Comment by sheabunge** [comment]
We're planning on introducing a Trash status, similar to how posts work. Would this help with this situation?

**Comment by paaljoachim** [comment]
Yes. That would help. As it would be a place to keep specific snippets out of the All Snippets screen.
I do wonder if there might be a need for an Archive screen....

I will go ahead and close and can always reopen if need be at a later time.
